public/views/question.html

```diff
@@ -35,3 +35,24 @@
     </div>
   </div>
 </div>
+
+<div id="message-modal" class="modal fade" role="dialog">
+  <div class="modal-dialog">
+    <!-- Modal content-->
+    <div class="modal-content">
+      <div class="modal-header">
+        <h4 class="modal-title" style="color: #800000">Diversity is Good, But Simplicity is the Best</h4>
+      </div>
+      <div class="modal-body">
+        <p>{{message}}</p><br>
+        <p>Kindly select a region&nbsp;&nbsp;
+        <select id="selectedCountry" style="text-align:center;" ng-model="selectedCountry" ng-options="region.country for region in countries" ng-change="selectCountry();"></select>
```

Comment:
rather than using ng-change here, use $scope.$watch('selectedCountry', () =>{}), that way we don't need to use querySelector